pxtsim/sound/soundexpression.ts

```diff
@@ -310,7 +312,7 @@ namespace pxsim.codal.music {
                     arr[i] = ((buff[i] - 512) / 512);
                 }
                 return arr;
-            }, synth.sampleRate, 0.03, () => cancelled || (isCancelled && isCancelled()))
+            }, synth.sampleRate, volume, () => cancelled || (isCancelled && isCancelled()))
```

Comment:
Should we pass in 0.03 if volume is undefined?

Reply:
If you do this, you could avoid the changes in SoundEffectEditor and SoundGallery since it'd just be the default, but it doesn't hurt to have those explicitly set the volume if you prefer it that way.